common/models/cert.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "cert",
+  "plural": "certs",
+  "base": "PersistedModel",
+  "idInjection": true,
+  "options": {
+    "validateUpsert": true
+  },
+  "properties": {
+    "name": {
+      "type": "string",
+      "required": true
+    },
+    "rfid": {
```

Comment:
This should be a list.

Reply:
So the argument for lists: one issue is that this isn't SQL or an RDBMS so the storage principles here don't really apply. This has two implications: the way you cross-reference and join records becomes a little more expensive, and we don't want to spend time thinking about optimizing storage under the hood; and having a single list to get/insert/delete from is going to be more resilient to bugs than a jumble of individiual records (unless we take advantage of loopback's schema constraints which we haven't so far). So both performance and data quality are kind of issues here, the latter being the one that we'll be more likely to have to spend time working on. Performance is also a concern but more for scaling.

At an app logic level, the precompute of the lists is important. Devices will be typically be requesting the whole list to cache.  e.g. `GET /api/certs/laser` should return:
```[{ "name": "laser", rfids: [x, y, z]}]```
This way clients don't have to be making tons of network calls and resolving them client side-- the precompute is primary-use-case-friendly and it gives clients flexibility.

I don't see the use case for individual listings. If we had a stronger object model, or were using an RDBMS, this might make sense but we would still want these lists cached and ready to be pulled.

Reply:
I see your points, all are valid. I'm ok with making a change to move these to lists - I should have time tonight to get that done so that anyone who wants to play with this over the holidays can do so.